doc/man3/SSL_CTX_get0_token_store.pod

```diff
@@ -0,0 +1,66 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_get0_token_store, SSL_CTX_set_token_store
+- QUIC NEW_TOKEN store manipulation 
+
+=head1 SYNOPSIS
+
+ SSL_TOKEN_STORE_HANDLE *SSL_CTX_get0_token_store(SSL_CTX *ctx);
+ int SSL_CTX_set_token_store(SSL_CTX *ctx, SSL_TOKEN_STORE_HANDLE *hdl);
+
+=head1 DESCRIPTION
+The QUIC protocol supports the exchange of opaque tokens which a client can use
+to reduce the time for a server to validate a client address.  These tokens are
```

Comment:
This sounds very much like SSL_SESSSION. Have we considered using that instead?

Reply:
I had, but tokens are representative of authorization/validation to a particular server, which may be used by multiple SSL_SESSIONS (and potentially between multiple SSL_CTX objects), which is why i chose to store them in the CTX, and allowed migrations between them via the get/set api.